pages/full.vue

```diff
@@ -28,13 +27,17 @@ const componentFileExists = computed<boolean>(() => {
 </script>
 
 <template>
-  <div v-if="componentFileExists">
-    <Component :is="component" />
-  </div>
-  <div v-else-if="item?.fullView">
-    <div v-html="item?.fullView"></div>
-  </div>
-  <div v-else>Item not found.</div>
+  <section class="section">
+    <div class="content is-size-5">
```

Comment:
Annoyingly I suggest not adding the `<section>` or `<div class="content">` here just because I know it'll mess up things that need full-bleed / picky-formatting controls rendered inside.  (The maps!)  This is kinda meh, because it means that the components that implement the Item will be carrying more formatting, but eh.

Reply:
That makes sense re: map formatting, etc. I've moved the section/content lines into the `ProjectedPermafrostAndMagt.vue` component and out of `full.vue` in d1f661f.